hstcal/meta.yaml

```diff
@@ -22,7 +22,6 @@ requirements:
     build:
       - cfitsio
       - pkg-config
-      - gcc >=4.7
```

Comment:
I'm not sure if this will work or is enough since we're still specifying the runtime lib as libgcc.

Reply:
Shouldn't we be following these [docs](https://conda.io/docs/user-guide/tasks/build-packages/compiler-tools.html) and using the following directive:
```yaml
requirements:
  build:
    - {{ compiler('c') }}
```

Reply:
I was having trouble having this work on a Mac and so had gone with a minimum suggestion to help build it.   As things currently stand, I'm going to close this PR as we should apply your fix and then go back to the original build, or figure out how this is really going to work with anaconda's currently available compilers.